### PR TITLE
Use latest versions

### PR DESCRIPTION
**Pull request by simonwahlgren**
Trying out this tag in integration for consumer-handler-service atm

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 REPO=fyndiq
 NAME=python-alpine-kafka
-TAG=python3.6.4-librdkafka0.11.4-v2
```
Comment (ghassen-telmoudi) [inline Makefile:3]: Why is this test?

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
-FROM python:3.6.4-alpine3.7
```
Comment (ghassen-telmoudi) [inline Dockerfile:1]: Have test our application with python version 3.7 first?

**Review by saabeilin — APPROVED** [review]
I've been testing it a bit a no new bugs :) So let's remove the `-test` in tag and start using it.

File: Makefile (hunk 1 above)
Comment (simonwahlgren) [reply Makefile:3]: Because it's not considered a stable image

File: Dockerfile (hunk 2 above)
Comment (simonwahlgren) [reply Dockerfile:1]: I have run consumer-handler-service and order-email-service with this image some weeks and so far everything looks good

**Pull request merged**